.github/workflows/download-translations.yml

```diff
@@ -0,0 +1,137 @@
+name: 'Download translations from Crowdin'
+
+permissions:
+  actions: write
+  checks: write
+  contents: write
+  deployments: write
+  pull-requests: write
+  statuses: write
+  
+  
+on:
+  repository_dispatch:
+    types:
+      - file-fully-reviewed
```

Comment:
This workflow will only trigger when a target language file has been fully approved, meaning that all translation strings should be approved, only then it will trigger.
We can choose to include another event where if a translation string is approved, this workflow will be triggered
cc: @yashim-deriv